docker/oso-saml-sso/centos7/run.sh

```diff
@@ -10,5 +10,5 @@
 
 
 echo -n "Running oso-centos7-saml-sso... "
-sudo docker run -ti --net=host --rm=true --name saml-sso oso-centos7-saml-sso $@
+docker run -ti --net=host --rm=true --name saml-sso oso-centos7-saml-sso $@
```

Comment:
This will prevent most people's configurations from running this image locally. Is there a reason it's different from the others or is this leftover config from testing?

Reply:
Thanks, Doug.  Yes, leftover from my testing.  I'll fix.